For some yet unknown reason, fixes exec crash at boot due to ext2fs.static
including this.

Index: glibc-2.24/libpthread/pthread/pt-exit.c
===================================================================
--- glibc-2.24.orig/libpthread/pthread/pt-exit.c
+++ glibc-2.24/libpthread/pthread/pt-exit.c
@@ -39,14 +39,14 @@ __pthread_exit (void *status)
   /* Run any cancelation handlers.  According to POSIX, the
      cancellation cleanup handlers should be called with cancellation
      disabled.  */
-  __pthread_setcancelstate (PTHREAD_CANCEL_DISABLE, &oldstate);
+  pthread_setcancelstate (PTHREAD_CANCEL_DISABLE, &oldstate);
 
   for (handlers = __pthread_get_cleanup_stack ();
        *handlers;
        *handlers = (*handlers)->__next)
     (*handlers)->__handler ((*handlers)->__arg);
 
-  __pthread_setcancelstate (oldstate, &oldstate);
+  pthread_setcancelstate (oldstate, &oldstate);
 
   /* Decrease the number of threads.  We use an atomic operation to
      make sure that only the last thread calls `exit'.  */
